### PR TITLE
refactor: stand-alone examples

### PR DESCRIPTION
**Pull request by IanChokS**
Refactored some old examples files (in which doing so fixed some old bugs), and added a couple of new example files.

### CODE REVIEW

**Comment by IanChokS** [comment]
Hi @arthurschreiber,

I tried looking into why the travis-ci test is failing but I am still unsure. Do you have any ideas?
Thanks

**Comment by arthurschreiber** [comment]
@IanChokS Could you update all examples to follow the coding style used for the rest of the codebase? I.e. run `npx eslint examples --fix`. 🙇

**Comment by IanChokS** [comment]
Done!

File: examples/TableValuedParameters.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+  const request = new Request(sql, (err, rowCount) => {
+    if (err) {
+      throw err;
+    } else {
+      console.log(`'${table}' created!`);
+    }
+  });
+
+  request.on('requestCompleted', () => {
```
Comment (arthurschreiber) [inline examples/TableValuedParameters.js]: I think it'd be better not to advertise the `requestCompleted` callback too much, but instead use the callback passed to `Request` instead. 🤔

Comment (IanChokS) [reply examples/TableValuedParameters.js]: I moved these 'requesCompleted' section into the Request call backs :)

**Comment by IanChokS** [comment]
@arthurschreiber I'm not exactly sure why codecov is failing the way it is. Do you have any ideas?

**Comment by IanChokS** [comment]
This is continued in #991 for a cleaner commit history

**Pull request closed without merging**